src/settings.ts

```diff
@@ -100,6 +98,10 @@ export const MAX_IMAGE_FILE_SIZE_TEXT = "2MB";
 export const MAX_AVATAR_IMAGE_FILE_SIZE_BYTES = 1024 * 150;
 export const GIF_IMAGE_WIDTH_PX = 300;
 
+// Tables
+export const MAX_TABLE_CAPACITY = 10;
+export const MIN_TABLE_CAPACITY = 2;
```

Comment:
[nit] Min before max.

```suggestion
export const MIN_TABLE_CAPACITY = 2;
export const MAX_TABLE_CAPACITY = 10;
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/commit/4b7532e6f02068126c212a6f0dab1714aaaad55a